oauth-function-template/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dronedeploy/oauth-function-template",
-  "version": "1.5.2",
+  "version": "1.6.0",
```

Comment:
Are we sure that this is a backward compatible change?

Reply:
Yes